cider/function/string/StringOps.cpp

```diff
@@ -228,8 +228,13 @@ NullableStrType Trim::operator()(const std::string& str) const {
     }
   }
   if (trim_begin == 0 && trim_end == str_len - 1) {
+    // no trimming should be applied
     return str;
   }
+  if (trim_begin >= trim_end) {
```

Comment:
will there be ```>``` condition? It seems only when str_len is 0, trim_end will be less than trim_begin. But this is returned in above code.

Reply:
Yes there will be. Let the query be `SELECT LTRIM('xxxxx', 'x')`. Then `trim_begin` will be `5` and `trim_end` will be `4`, resulting in `trim_begin > trim_end` and `trim_begin > str_len - 1`.